### PR TITLE
Remove self profile, move profile to settings

### PR DESCRIPTION
**Pull request by mikeger**
# Issue
The other mobile clients are moving to use the user profile integrated to the settings. We need to catch up in order to be able to start with other features that require richer settings. Also with the new settings the navigation to the devices list is 2 clicks closer to user.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController.m [hunk 1]
```diff
@@ -879,9 +888,34 @@ - (void)conversationListBottomBar:(ConversationListBottomBarController *)bar did
             break;
             
         case ConversationListButtonTypeSettings:
-            [[ZClientViewController sharedZClientViewController] openSelfProfileAnimated:YES];
-            break;
+        {
+            SettingsNavigationController *settingsViewController = [self createSettingsViewController];
             
+            if (IS_IPAD) {
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController.m]: I suspect this won't work well with split screen when in iphone size.

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/ConversationList/ConversationListViewController.m]: Right! I updated to check with traits

File: Wire-iOS/Sources/UserInterface/Helpers/UIFont+MonoSpaced.swift [hunk 2]
```diff
@@ -28,6 +28,20 @@ private let monospaceAttribute = [
     UIFontDescriptorFeatureSettingsAttribute: [monospacedFeatureSettingsAttribute]
 ]
 
+private let allCapsGeatureSettingsAttributeLowerCase = [
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Helpers/UIFont+MonoSpaced.swift]: Typo `Geature` → `Feature`

File: Wire-iOS/Sources/UserInterface/Helpers/UIFont+MonoSpaced.swift [hunk 3]
```diff
@@ -28,6 +28,20 @@ private let monospaceAttribute = [
...
+    UIFontFeatureTypeIdentifierKey: kLowerCaseType,
+    UIFontFeatureSelectorIdentifierKey: kLowerCaseSmallCapsSelector,
+]
+
+private let allCapsGeatureSettingsAttributeUpperCase = [
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Helpers/UIFont+MonoSpaced.swift]: Typo `Geature` → `Feature`

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Helpers/UIFont+MonoSpaced.swift]: Updated! Thanks

**Pull request merged**